### PR TITLE
LFX Mentorship - OpenTelemetry - Mar 2024

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -122,6 +122,25 @@
...
+- Recommended Skills: One (or more!) of the missing languages supported by OpenTelemetry
+  - C++
+  - Erlang
+  - Go
+  - JavaScript
+  - PHP
+  - Ruby
+  - Swift
+- Mentor(s):
+  - Juraci Paixão Kröhling (@jpkrohling, juraci.kroehling@grafana.com)
```
Comment (aliok) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:140]: We strongly encourage mentors to have a backup mentor. Do you think you can find one?

**Comment by aliok** [comment]
Thanks @jpkrohling .

As I wrote above, we strongly encourage mentors to have a backup mentor for coverage of any unplanned issues throughout the mentoring program. Do you think you can find a co-mentor?

I will keep this PR open for your response.

Also, I wanted to let you know that this term we’re doing an information session for mentors. We’d like to invite all the mentors participating to attend one of these if you're able:
* North America: [Jan 29, 2024 10:30 AM (Pacific Time)](https://zoom-lfx.platform.linuxfoundation.org/meeting/99504894798?password=54a6ec2f-d649-432b-a918-acd17720f163)
* EMEA + APAC: [Jan 30, 2024 8:00 AM (UTC)](https://zoom-lfx.platform.linuxfoundation.org/meeting/93514971498?password=907d67b6-3e04-42d3-afdc-52d288122c6f)

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md (hunk 1 above)
Comment (jpkrohling) [reply programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:140]: Yes!

**Review by astencel-sumo — APPROVED** [review]
Thanks Juraci

**Review by aliok — APPROVED** [review]

**Pull request merged**